lib/src/sql/common.rs

```diff
@@ -112,6 +112,14 @@ pub fn take_u64(i: &str) -> IResult<&str, u64> {
 	}
 }
 
+pub fn take_u32(i: &str) -> IResult<&str, u32> {
+	let (i, v) = take_while(is_digit)(i)?;
+	match v.parse::<u32>() {
+		Ok(v) => Ok((i, v)),
+		_ => Err(Error(Parser(i))),
+	}
+}
+
```

Comment:
You may use the u32 function from complete:

```rust
    use nom::character::complete::u32;
```

Reply:
Wow, thanks! Replaced it with the nom one in b924b936!